shoes-core/lib/shoes/common/state.rb

```diff
@@ -1,16 +1,30 @@
 class Shoes
   module Common
     module State
```

Comment:
Cool that you dug that up!

I guess the reasoning was most methods are auto generated by style, which is true. But now we needed more stuff to reinstate it - software always changes and it is not uncommon to go back and forth on a decision (at least in my experience) :)